packages/components/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@
 -   `Sandbox`: Use `toString` to create observe and resize script string ([#42872](https://github.com/WordPress/gutenberg/pull/42872)).
 -   `Navigator`: refactor unit tests to TypeScript and to `user-event` ([#44970](https://github.com/WordPress/gutenberg/pull/44970)).
 -   `Navigator`: Refactor Storybook code to TypeScript and controls ([#44979](https://github.com/WordPress/gutenberg/pull/44979)).
+-   `ToolsPanel`: updated to satisfy `react/exhaustive-deps` eslint rule ([#45028](https://github.com/WordPress/gutenberg/pull/45028))
```

Comment:
After merging #45043, I realized that its CHANGELOG entry didn't get added to the `Unreleased` section — would you be able to move that entry just under this one? Thank you!

Reply:
oops! nice catch, I'm glad you spotted it. I've rebased this PR and moved the `Tooltip` CHANGELOG entry up to `Unreleased` 👍